modules/client-configuration/pages/delete-clients.adoc

```diff
@@ -0,0 +1,18 @@
+[[delete.clients]]
+= Delete Clients
+
+If you need to remove a client from your {productanme} Server, you can use the {webui} to delete it.
+This procedure works for both traditional and Salt clients, however Salt clients require extra steps to clean up configuration files after the client is deleted.
+
+
+
+.Procedure: Deleting a Client
+. In the {productname} {webui}, navigate to menu:Systems[System List] and select the client to delete.
+. Click btn:[Delete System].
+. Check the details and click btn:[Delete Profile] to confirm.
+. For Salt clients, {productname} attempts to clean up additional configuration.
+  If the client can not be contacted, you can cancel the deletion, or delete the client without cleaning up the configuration files.
```

Comment:
```suggestion
  If the client cannot be contacted, you can cancel the deletion, or delete the client without cleaning up the configuration files.
```
This sounds strange.  Is this actually a synchronous action?

Reply:
"you are given the option to"?